src/Tribe/Aggregator/API/Image.php

```diff
@@ -67,16 +67,20 @@ public function get( $image_id ) {
 			return $response;
 		}
 
+		// Get Headers and Body
+		$headers = wp_remote_retrieve_headers( $response );
+		$body = wp_remote_retrieve_body( $response );
+
```

Comment:
Probably too much of a defense but: we are making assumptions about the `$headers` array wishing any key we look for will be there (e.g. `content-type` and `content-disposition`); a lot of filtering could happen in that request and I'd advise a pre-emptive check on the keys we look for before trying to look them up.
Since the `Requests_Utility_CaseInsensitiveDictionary` class implements `ArrayAccess` use of an `array_intersect_key` is fast and fine here like:

```php
$go = count(array_intersect_key(array('content-type', 'content-disposition'), $headers)) === 2
```